bitnami/kibana/templates/deployment.yaml

```diff
@@ -0,0 +1,186 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "kibana.fullname" . }}
+  labels: {{- include "kibana.labels" . | nindent 4 }}
+spec:
+  replicas: {{ .Values.replicaCount }}
+  {{- if .Values.updateStrategy }}
+  strategy: {{- tpl (toYaml .Values.updateStrategy) $ | nindent 4 }}
+  {{- end }}
+  selector:
+    matchLabels: {{- include "kibana.matchLabels" . | nindent 6 }}
+  template:
+    metadata:
+      labels: {{- include "kibana.labels" . | nindent 8 }}
+    spec:
+    {{- if .Values.schedulerName }}
+      schedulerName: {{ .Values.schedulerName | quote }}
+    {{- end }}
+{{- include "kibana.imagePullSecrets" . | indent 6 }}
+    {{- if .Values.securityContext.enabled }}
+      securityContext:
+        fsGroup: {{ .Values.securityContext.fsGroup }}
+    {{- end }}
+    {{- if or .Values.initContainers (and .Values.volumePermissions.enabled .Values.persistence.enabled) }}
+      initContainers:
+      {{- if and .Values.volumePermissions.enabled .Values.persistence.enabled }}
+      - name: volume-permissions
+        image: "{{ template "kibana.volumePermissions.image" . }}"
+        imagePullPolicy: {{ default "" .Values.volumePermissions.image.pullPolicy | quote }}
+        command: ["chown", "-R", "{{ .Values.securityContext.runAsUser }}:{{ .Values.securityContext.fsGroup }}", "{{ .Values.persistence.mountPath }}"]
+        securityContext:
+          runAsUser: 0
+        resources: {{ toYaml .Values.volumePermissions.resources | nindent 10 }}
+        volumeMounts:
+        - name: data
+          mountPath: {{ .Values.persistence.mountPath }}
+      {{- end }}
+      {{- if .Values.initContainers }}
+      {{- tpl (toYaml .Values.initContainers) $ | nindent 6 }}
+      {{- end }}
+    {{- end }}
+      containers:
+        - name: kibana
+          image: {{ include "kibana.image" . }}
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          {{- if .Values.securityContext.enabled }}
+          securityContext:
+            runAsUser: {{ .Values.securityContext.runAsUser }}
+          {{- end }}
+          env:
+            - name: KIBANA_ELASTICSEARCH_URL
+              value: {{ include "kibana.elasticsearch.url" . | quote }}
+            - name: KIBANA_ELASTICSEARCH_PORT
+              value: {{ include "kibana.elasticsearch.port" . | quote }}
+            - name: KIBANA_FORCE_INITSCRIPTS
```

Comment:
should `KIBANA_FORCE_INITSCRIPTS` be `KIBANA_FORCE_INIT_SCRIPTS`?

Reply:
Well, in the container I put KIBANA_FORCE_INITSCRIPTS

https://github.com/bitnami/bitnami-docker-kibana/blob/bdf9cdd7aeda914e83a9fbaf9d8813568375e951/7/debian-9/rootfs/run.sh#L25

We would need to change it in the container as well

Reply:
ok then